oeps/architectural-decisions/oep-0052/decisions/002-schema-specification.rst

```diff
@@ -0,0 +1,30 @@
+Event Bus Messages will be Defined in AVRO Specification Language
+=================================================================
+
+Context
+-------
+
+* Currently, Pulsar does not support `JSON Schema <https://json-schema.org/specification.html>`__, it only supports AVRO
+  for schema definition. Pulsar does support serializing AVRO to JSON (confusing using a ``JSONSchema`` class).
+
+* Industry best practices seem to suggest using a binary encoding of messages.(TODO, links)
```

Comment:
@feanil Do you by any chance have the `TODO links` mentioned in this ADR? I haven't been able to find authoritative  sources for these statements.

Reply:
I don't have them on-hand no.